source/releasenotes/2024-12-10-composer-2-8-3.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Composer 2.8.3 now available
+published_date: "2024-12-10"
+categories: [infrastructure,tools-apis]
+---
+
+Composer 2.8.3 is now available on Pantheon as of December 10, 2024.
+
+### No action required
+
+Composer 2.8.3 will slowly roll out to all Integrated Composer sites on Pantheon automatically. There is no action required on your part.
+
```

Comment:
```suggestion
Composer commands can be run against sites on Pantheon via a [Terminus plugin](https://github.com/pantheon-systems/terminus-composer-plugin). For operations that modifying version controlled code (like adding a module via `composer require` which will change `composer.json` and `composer.lock`) you must run such commands in Dev or Multidev environments where version controlled files are writable. Such files are not writeable in Test and Live environments.

```

Reply:
@stevector 

When would I use Terminus to `composer require` a thing rather than `composer require`ing locally and pushing the updated `composer.json`? Is that process different?

Reply:
Also: is this note relevant to the 2.8.3 _update_ to Composer specifically? I don't know that we should be adding this level of detail outside the documentation for Composer on the platform unless we're going to copy/paste this with every update to Composer. I think linking to the Composer docs should be sufficient.